src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -15,20 +15,26 @@ const WAVEFROM_SCALE_FACTOR = 5
 let recordingLength = 0;
 let recordBuffer = [[], []];
 let isRecording = false;
-let isMonitoring = false;
-let visualizationEnabled = true;
+let initCount = 0;
 
 let recordButton = document.querySelector('#record');
 let recordText = document.querySelector('#record-text');
+let stopButton = document.querySelector('#stop');
 let player = document.querySelector('#player');
 let downloadButton = document.querySelector('#download-button');
 let downloadLink = document.querySelector('#download-link');
 
 // Wait for user interaction to initialize audio, as per specification.
-document.addEventListener('click', (element) => {
-  init();
-  document.querySelector('#click-to-start').remove();
-}, {once: true});
+if (!initCount){
+  recordButton.disabled = false;
+  recordButton.addEventListener('click', (element) => {
+    init();
+    isRecording = true;
+    initCount++;
```

Comment:
Why do we need counting? Can't this be `_isInitialized` boolean flag?

Reply:
I make the count here instead of the boolean is that I want to try my best keep the SPN recorder and worklet recorder using the same parameter so the developer can have a better and clear compare the difference between these two method.

Reply:
See my response below.